### ISSUE TITLE
snap package

### ISSUE THREAD

**Issue by pixelr0** [openstreetmap/merkaartor#230]
someone just packaged Merkaartor as a snap. you can see the snap here https://snapcraft.io/merkaartor-james-carroll
snaps work across Linux on any distribution or version. you can add it to your download page if you want http://merkaartor.be/p/download

Merkaartor snap source is here https://github.com/mrcarroll/merkaartor-snap (if you want to make an official snap and don't know how)

**Comment by KAMiKAZOW** [comment]
> snaps work across Linux on any distribution or version

That's not true. For example, openSUSE does not ship Snap in the default repository because the openSUSE build breaks all the time. Gentoo, Mandriva, Mageia, and many others don't get a port at all and distributions that get a port of Snap usually have several restrictions.

I'm not trying to discourage the inclusion on the download page. For Ubuntu users it's certainly a good way to get Merkaartor outside of distribution release cycles but it shouldn't be advertised as something that just works everywhere.

**Comment by MrCarroll** [comment]
I've published the snap under a new package name, not that it increases officialness, bit it looks nicer and makes it simpler to transfer ownership in the event I step down. ( So people aren't stuck with my name forever ;) )

https://snapcraft.io/merkaartor

The old package is temporarily still available while the rest of its users recieve the 0.19.0 update, but in a few days I'll prevent new downloads. Because there was only just over a hundred or so active users of the older package combined with the release cadence of Merkaartor, I don't expect this transition to cause much confusion. However if people do see this, consider `sudo snap remove merkaartor-james-carroll` and `sudo snap install merkaartor` to migrate over fully.
